### DISCUSSION TITLE
Develco SIRZB-110 or other siren

### DISCUSSION THREAD

**Discussion by Grazulex** [Koenkk/zigbee2mqtt#16843] [Devices]
Hello,

I have buy  a Develco SIRZB-110 but it's impossible to use it.
Pairing was ok, but when I try to send command to this device, I have a lot of "no response" issue...and the command "system_is_armed" and "system_is_disarmed" are not reconnize too....
Have somebody the good way to used it or a ref of a other zigbee siren (more simple) ?

thanks

**Comment by lux73** [comment]
it is marked as supported?

latest z2m installed?

**Reply by Grazulex** [reply]
Yes (this device is in the list of the supported one) and yes I have try with the last master version of z2m and the dev one.

**Reply by lux73** [reply]
Thats not what i mean

Please look in the WebIf if its marked as supported

Sometimes the manufacturer changes the inernal Device Name so this particular Device is not more recognized from z2m and statet as unsupported

**Reply by Grazulex** [reply]
I have check : Supported

**Comment by Grazulex** [comment]
For you info, after the last update, the siren is working well now.

**Reply by lux73** [reply]
Nice to hear 👍

**Comment by sigalou** [comment]
Hello, I'm stuck with this siren
https://www.zigbee2mqtt.io/devices/SIRZB-110.html
The documentation specifies:

> Only works with firmware 1.7.1 and lower (whatever hardware version). Firmware version 1.9.3 uses a different framework and these commands no longer work.

I don't plan to change the firmware version to an old version, it's DIY troubleshooting, it's not a solution.
I read your discussion above, what firmware version do you have?
Has anyone planned to modify pushing mqtt so that it works with the new version?
THANKS

**Reply by sigalou** [reply]
@Grazulex @lux73
